settings.py

```diff
@@ -367,12 +367,10 @@
     'luna',
     'raven.contrib.django.raven_compat',
     'rosetta',
-    'soil',
     'custom.apps.crs_reports',
     'custom.m4change',
 
     # submodules with tests that run on travis
-    # 'couchexport',
```

Comment:
Haha I think I remember trying to exclude this at some point and that interacting poorly with other tests... so no change necessary I guess